<!DOCTYPE html>
<!-- saved from url=(0045)https://swissmodel.expasy.org/docs/references -->
<html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><!-- Global site tag (gtag.js) - Google Analytics --><script type="text/javascript" async="" src="../static/js/analytics.js"></script><script async="" src="../static/js/js"></script><script>
      window.dataLayer = window.dataLayer || [];
      function gtag(){dataLayer.push(arguments);}
      gtag('js', new Date());

      gtag('config', 'UA-8667194-2');
    </script><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width, initial-scale=1"><meta name="ROBOTS" content="ALL"><title>References | SWISS-MODEL</title><meta name="description" content="SWISS-MODEL is a fully automated protein structure homology-modelling server. The purpose of this server is to make protein modelling accessible to all life science researchers worldwide."><link rel="stylesheet" type="text/css" href="../static/css/bootstrap-3.3.7.min.css" media="screen,print"><link rel="stylesheet" type="text/css" href="../static/css/bootstrap-theme-3.3.7.min.css" media="screen,print"><link rel="stylesheet" type="text/css" href="../static/css/master.css" media="all"><link rel="stylesheet" type="text/css" href="../static/css/jquery.fancybox.css"><link rel="stylesheet" type="text/css" href="../static/css/jquery-ui.min.css"><link rel="shortcut icon" href="https://swissmodel.expasy.org/static/images/favicon.png" type="image/png"><link rel="apple-touch-icon" href="https://swissmodel.expasy.org/static/images/apple-touch-57.png"><link rel="apple-touch-icon" sizes="72x72" href="https://swissmodel.expasy.org/static/images/apple-touch-72.png"><link rel="apple-touch-icon" sizes="114x114" href="https://swissmodel.expasy.org/static/images/apple-touch-114.png"><style>
  .list-group-item{
    border:unset;
    padding:8px 24px;
  }
  .list-group {
    -webkit-box-shadow: unset;
    box-shadow: unset;
}
</style></head><body><nav class="navbar navbar-default"><div class="container-fluid"><div class="navbar-header"><button aria-controls="navbar" aria-expanded="false" data-target=".collapseThis" data-toggle="collapse" class="navbar-toggle collapsed" type="button"><span class="sr-only">Toggle navigation</span><span class="icon-bar"></span><span class="icon-bar"></span><span class="icon-bar"></span></button><a class="hidden-xs hidden-sm navbar-brand" style="margin-left:0px;padding:8px 4px 0px 0px;" title="SIB Swiss Institute of Bioinformatics" href="https://www.sib.swiss/" target="_blank"><img src="../static/img/sib_logo_small.png" alt="SIB logo" style="max-height:50px"></a><a class="hidden-xs hidden-sm navbar-brand" style="margin-left:0px; padding:8px 0px 0px 4px;" title="Biozentrum | University of Basel | The Center for Molecular Life Sciences" href="http://www.biozentrum.unibas.ch/" target="_blank"><img src="../static/img/biozentrum_en.png" alt="Biozentrum logo" style="max-height:36px"></a><a class="hidden-xs hidden-sm navbar-brand" href="https://swissmodel.expasy.org/" style="margin-left:0px;padding-left:0px;padding-right:0px;font-size:28px;">SWISS-MODEL</a><a id="navbar-brand" class="hidden-md hidden-lg navbar-brand" href="https://swissmodel.expasy.org/" style="margin-left:0px;font-size:20px;">SWISS-MODEL</a></div><div id="navbar" class="collapseThis navbar-collapse collapse"><!-- Collect the nav links, forms, and other content for toggling --><ul class="nav navbar-nav navbar-right"><li class="dropdown "><a href="https://swissmodel.expasy.org/interactive" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Modelling</a><ul class="dropdown-menu" role="menu"><li><a class="text-left" href="https://swissmodel.expasy.org/interactive">myWorkspace</a></li><li><a class="text-left" href="https://swissmodel.expasy.org/interactive#alignment">Alignment Mode</a></li><li><a class="text-left" href="https://swissmodel.expasy.org/interactive#structure">User Template Mode</a></li><li><a class="text-left" href="https://swissmodel.expasy.org/interactive#deepview">DeepView Project Mode</a></li><li class="divider"></li><li><a class="text-left" href="https://swissmodel.expasy.org/templates/">Template Library</a></li></ul></li><li id="smrLink" class="dropdown "><a href="https://swissmodel.expasy.org/repository" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Repository</a><ul class="dropdown-menu" role="menu"><li><a href="https://swissmodel.expasy.org/repository">SWISS-MODEL Repository</a></li><li><a href="https://swissmodel.expasy.org/repository/user_annotation_upload">Annotation Projects</a></li></ul></li><li class="dropdown "><a class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Tools</a><ul class="dropdown-menu" role="menu"><li><a href="https://swissmodel.expasy.org/qmean/" tabindex="-1">QMEAN</a></li><li><a href="https://swissmodel.expasy.org/lddt" tabindex="-1">lDDT</a></li><li><a href="https://swissmodel.expasy.org/assess">Structure Assessment</a></li><li><a href="https://swissmodel.expasy.org/comparison/">Structure Comparison</a></li></ul></li><li class="dropdown active"><a href="https://swissmodel.expasy.org/docs/references#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Documentation</a><ul class="dropdown-menu" role="menu"><li><a href="https://swissmodel.expasy.org/docs/help" tabindex="-1">SWISS-MODEL</a></li><li><a href="https://swissmodel.expasy.org/docs/examples" tabindex="-1">Examples</a></li><li><a href="https://swissmodel.expasy.org/docs/tutorial" tabindex="-1">Video Tutorial</a></li><li><a href="https://swissmodel.expasy.org/docs/blog" tabindex="-1">New Features</a></li><li role="separator" class="divider"></li><li><a href="https://swissmodel.expasy.org/docs/repository_help" tabindex="-1">SWISS-MODEL Repository</a></li><li role="separator" class="divider"></li><li><a href="https://swissmodel.expasy.org/qmean/help" tabindex="-1">QMEAN</a></li><li><a href="https://swissmodel.expasy.org/lddt/help/" tabindex="-1">lDDT</a></li><li><a href="https://swissmodel.expasy.org/assess/help">Structure Assessment</a></li><li><a href="https://swissmodel.expasy.org/comparison/help">Structure Comparison</a></li><li role="separator" class="divider"></li><li><a href="https://swissmodel.expasy.org/course" tabindex="-1">Protein Structure Course</a></li><li role="separator" class="divider"></li><li><a href="https://swissmodel.expasy.org/docs/references" tabindex="-1">References</a></li></ul></li><li><a onclick="showLogin();">Log in</a></li><li><a onclick="createAccount();">Create Account</a></li></ul></div></div><!-- /container-fluid --></nav><div id="tools"></div><div id="main-content" style="padding-left:32px; margin-right:32px"><a name="references"></a><h3>References</h3><p><b>If you are using models from the <i>SWISS-MODEL Server or Repository</i>,
      please cite the corresponding articles:</b></p><ul><li><a name="swissmodel" id="swissmodel"></a><b>SWISS-MODEL Workspace/ GMQE</b><br>
              Waterhouse, A., Bertoni, M., Bienert, S., Studer, G., Tauriello, G., Gumienny, R.,
              Heer, F.T., de Beer, T.A.P., Rempfer, C., Bordoli, L., Lepore, R., Schwede, T.    
              SWISS-MODEL: homology modelling of protein structures and complexes.
              Nucleic Acids Res. 46, W296-W303 (2018).
              <a href="https://pubmed.ncbi.nlm.nih.gov/29788355" target="_blank"><img src="../static/img/pubmed.png" width="17" height="16" align="absmiddle"></a><a href="https://doi.org/10.1093/nar/gky427" target="_blank"><img src="../static/css/doi.jpg" width="30" height="16" align="absmiddle" style="padding-left:2px"></a></li><li><b>SWISS-MODEL Repository</b><br>
          Bienert, S., Waterhouse, A., de Beer, T.A.P., Tauriello, G., Studer, G.,
          Bordoli, L., Schwede, T. The SWISS-MODEL Repository - new features
          and functionality. Nucleic Acids Res. 45, D313-D319 (2017).
          <a href="https://pubmed.ncbi.nlm.nih.gov/27899672" target="_blank"><img src="../static/img/pubmed.png" width="17" height="16" align="absmiddle"></a><a href="https://dx.doi.org/10.1093/nar/gkw1132" target="_blank"><img src="../static/css/doi.jpg" width="30" height="16" align="absmiddle" style="padding-left:2px"></a></li><li><a name="deepview"></a><b>Swiss-PdbViewer/ DeepView project mode</b><br>
          Guex, N., Peitsch, M.C., Schwede, T. Automated comparative protein
          structure modeling with SWISS-MODEL and Swiss-PdbViewer: A historical
          perspective. Electrophoresis 30, S162-S173 (2009).
          <a href="https://pubmed.ncbi.nlm.nih.gov/19517507" target="_blank"><img src="../static/img/pubmed.png" width="17" height="16" align="absmiddle"></a><a href="https://dx.doi.org/10.1002/elps.200900140" target="_blank"><img src="../static/css/doi.jpg" width="30" height="16" align="absmiddle" style="padding-left:2px"></a></li><li><a name="promod3"></a><b>ProMod3</b><br>
          Studer, G., Tauriello, G., Bienert, S., Biasini, M., Johner, N.,
          Schwede, T. ProMod3 - A versatile homology modelling toolbox. 
          PLOS Comp. Biol. 17(1), e1008667 (2021).
          <a href="https://pubmed.ncbi.nlm.nih.gov/33507980" target="_blank"><img src="../static/img/pubmed.png" width="17" height="16" align="absmiddle"></a><a href="https://doi.org/10.1371/journal.pcbi.1008667" target="_blank"><img src="../static/css/doi.jpg" width="30" height="16" align="absmiddle" style="padding-left:2px"></a></li><li><a name="qmean"></a><b>QMEANDisCo</b><br>
          Studer, G., Rempfer, C., Waterhouse, A.M., Gumienny, R., Haas, J., 
          Schwede, T. QMEANDisCo - distance constraints applied on model quality 
          estimation. Bioinformatics 36, 1765-1771 (2020).
          <a href="https://pubmed.ncbi.nlm.nih.gov/31697312" target="_blank"><img src="../static/img/pubmed.png" width="17" height="16" align="absmiddle"></a><a href="https://dx.doi.org/10.1093/bioinformatics/btz828" target="_blank"><img src="../static/css/doi.jpg" width="30" height="16" align="absmiddle" style="padding-left:2px"></a></li><li><a name="qmeanzscore"></a><b>QMEAN</b><br>
          Benkert, P., Biasini, M., Schwede, T. Toward the estimation 
          of the absolute quality of individual protein structure 
          models. Bioinformatics 27, 343-350 (2011).
          <a href="https://pubmed.ncbi.nlm.nih.gov/21134891" target="_blank"><img src="../static/img/pubmed.png" width="17" height="16" align="absmiddle"></a><a href="https://dx.doi.org/10.1093/bioinformatics/btq662" target="_blank"><img src="../static/css/doi.jpg" width="30" height="16" align="absmiddle" style="padding-left:2px"></a></li><li><a name="qsqe"></a><b>Quaternary Structure Prediction/ QSQE</b><br>
          Bertoni, M., Kiefer, F., Biasini, M., Bordoli, L., Schwede, T.
          Modeling protein quaternary structure of homo- and hetero-oligomers
          beyond binary interactions by homology. Scientific Reports 7 (2017).
          <a href="https://pubmed.ncbi.nlm.nih.gov/28874689" target="_blank"><img src="../static/img/pubmed.png" width="17" height="16" align="absmiddle"></a><a href="https://dx.doi.org/10.1038/s41598-017-09654-8" target="_blank"><img src="../static/css/doi.jpg" width="30" height="16" align="absmiddle" style="padding-left:2px"></a></li><li><a name="lddt"></a><b>lDDT</b><br>
          Mariani, V., Biasini, M., Barbato, A., Schwede, T. lDDT: a
          local superposition-free score for comparing protein
          structures and models using distance difference tests.
          Bioinformatics 29, 2722-2728 (2013). 
          <a href="https://pubmed.ncbi.nlm.nih.gov/23986568" target="_blank"><img src="../static/img/pubmed.png" width="17" height="16" align="absmiddle"></a><a href="https://dx.doi.org/10.1093/bioinformatics/btt473" target="_blank"><img src="../static/css/doi.jpg" width="30" height="16" align="absmiddle" style="padding-left:2px"></a></li><!--END OF ORIGINAL LIST--><li><div class="panel-group"><div class="panel"><div><a data-toggle="collapse" href="https://swissmodel.expasy.org/docs/references#collapse1"><b>Click for a
                    chronological assembly of SWISS-MODEL publications.</b></a></div><div id="collapse1" class="panel-collapse collapse"><ul class="list-group"><li class="list-group-item">
                    Studer, G., Tauriello, G., Bienert, S., Waterhouse, A.M.,
                    Bertoni, M., Bordoli, L., Schwede, T., Lepore, R. Modeling
                    of Protein Tertiary and Quaternary Structures Based on
                    Evolutionary Information. Methods Mol. Biol. 1851, 301-316
                    (2019).
                    <a href="https://pubmed.ncbi.nlm.nih.gov/30298405" target="_blank"><img src="../static/img/pubmed.png" width="17" height="16" align="absmiddle"></a><a href="https://dx.doi.org/10.1007/978-1-4939-8736-8_17" target="_blank"><img src="../static/css/doi.jpg" width="30" height="16" align="absmiddle" style="padding-left:2px"></a></li><li class="list-group-item">
                    Biasini, M., Bienert, S., Waterhouse, A., Arnold, K.,
                    Studer, G., Schmidt, T., Kiefer, F., Cassarino, T.G.,
                    Bertoni, M., Bordoli, L., Schwede, T. SWISS-MODEL: modelling
                    protein tertiary and quaternary structure using evolutionary
                    information. Nucleic Acids Res. 42, W252-W258 (2014).
                    <a href="https://pubmed.ncbi.nlm.nih.gov/24782522" target="_blank"><img src="../static/img/pubmed.png" width="17" height="16" align="absmiddle"></a><a href="https://dx.doi.org/10.1093/nar/gku340" target="_blank"><img src="../static/css/doi.jpg" width="30" height="16" align="absmiddle" style="padding-left:2px"></a></li><li class="list-group-item">
                    Studer, G., Biasini, M., Schwede, T. Assessing the local
                    structural quality of transmembrane protein models using
                    statistical potentials (QMEANBrane). Bioinformatics 30,
                    i505-11 (2014). 
                    <a href="https://pubmed.ncbi.nlm.nih.gov/25161240" target="_blank"><img src="../static/img/pubmed.png" width="17" height="16" align="absmiddle"></a><a href="https://dx.doi.org/10.1093/bioinformatics/btu457" target="_blank"><img src="../static/css/doi.jpg" width="30" height="16" align="absmiddle" style="padding-left:2px"></a></li><li class="list-group-item">
                    Mariani, V., Biasini, M., Barbato, A., Schwede, T. lDDT: a
                    local superposition-free score for comparing protein
                    structures and models using distance difference tests.
                    Bioinformatics 29, 2722-2728 (2013). 
                    <a href="https://pubmed.ncbi.nlm.nih.gov/23986568" target="_blank"><img src="../static/img/pubmed.png" width="17" height="16" align="absmiddle"></a><a href="https://dx.doi.org/10.1093/bioinformatics/btt473" target="_blank"><img src="../static/css/doi.jpg" width="30" height="16" align="absmiddle" style="padding-left:2px"></a></li><li class="list-group-item">
                    Bordoli, L., Schwede T. Automated protein structure
                    modeling with SWISS-MODEL Workspace and the Protein Model
                    Portal. Methods Mol. Biol. 857, 107-136 (2012).
                    <a href="https://pubmed.ncbi.nlm.nih.gov/22323219" target="_blank"><img src="../static/img/pubmed.png" width="17" height="16" align="absmiddle"></a><a href="https://dx.doi.org/10.1007/978-1-61779-588-6_5" target="_blank"><img src="../static/css/doi.jpg" width="30" height="16" align="absmiddle" style="padding-left:2px"></a></li><li class="list-group-item">
                    Bordoli, L., Kiefer, F., Arnold, K., Benkert, P.,
                    Battey, J., Schwede, T. Protein structure homology modeling
                    using SWISS-MODEL workspace. Nat. Protoc. 4, 1–13 (2009).
                    <a href="https://pubmed.ncbi.nlm.nih.gov/19131951" target="_blank"><img src="../static/img/pubmed.png" width="17" height="16" align="absmiddle"></a><a href="https://dx.doi.org/10.1038/nprot.2008.197" target="_blank"><img src="../static/css/doi.jpg" width="30" height="16" align="absmiddle" style="padding-left:2px"></a></li><li class="list-group-item">
                    Kiefer, F., Arnold, K., Kuenzli, M., Bordoli, L.,
                    Schwede, T. The SWISS-MODEL Repository and associated
                    resources. Nucleic Acids Res. 37, D387–D392 (2009).
                    <a href="https://pubmed.ncbi.nlm.nih.gov/18931379" target="_blank"><img src="../static/img/pubmed.png" width="17" height="16" align="absmiddle"></a><a href="https://dx.doi.org/10.1093/nar/gkn750" target="_blank"><img src="../static/css/doi.jpg" width="30" height="16" align="absmiddle" style="padding-left:2px"></a></li><li class="list-group-item">
                    Benkert, P., Künzli, M., Schwede, T. QMEAN server for
                    protein model quality estimation. Nucleic Acids Res. 37,
                    W510-W514 (2009).
                    <a href="https://pubmed.ncbi.nlm.nih.gov/19429685" target="_blank"><img src="../static/img/pubmed.png" width="17" height="16" align="absmiddle"></a><a href="https://dx.doi.org/10.1093/nar/gkp322" target="_blank"><img src="../static/css/doi.jpg" width="30" height="16" align="absmiddle" style="padding-left:2px"></a></li><li class="list-group-item">
                    Benkert, P., Schwede, T., Tosatto, S.C.E. QMEANclust:
                    estimation of protein model quality by combining a
                    composite scoring function with structural density
                    information. BMC Struct. Biol. 9, 35 (2009).
                    <a href="https://pubmed.ncbi.nlm.nih.gov/19457232" target="_blank"><img src="../static/img/pubmed.png" width="17" height="16" align="absmiddle"></a><a href="https://dx.doi.org/10.1186/1472-6807-9-35" target="_blank"><img src="../static/css/doi.jpg" width="30" height="16" align="absmiddle" style="padding-left:2px"></a></li><li class="list-group-item">
                    Arnold, K., Bordoli, L., Kopp, J., Schwede, T. The
                    SWISS-MODEL workspace: a web-based environment for protein
                    structure homology modelling. Bioinformatics 22, 95-201
                    (2006).
                    <a href="https://pubmed.ncbi.nlm.nih.gov/16301204" target="_blank"><img src="../static/img/pubmed.png" width="17" height="16" align="absmiddle"></a><a href="https://dx.doi.org/10.1093/bioinformatics/bti770" target="_blank"><img src="../static/css/doi.jpg" width="30" height="16" align="absmiddle" style="padding-left:2px"></a></li><li class="list-group-item">
                    Kopp, J., Schwede, T. The SWISS-MODEL Repository: new
                    features and functionalities. Nucleic Acids Res. 34,
                    D315–D318 (2006).
                    <a href="https://pubmed.ncbi.nlm.nih.gov/16381875" target="_blank"><img src="../static/img/pubmed.png" width="17" height="16" align="absmiddle"></a><a href="https://dx.doi.org/10.1093/nar/gkj056" target="_blank"><img src="../static/css/doi.jpg" width="30" height="16" align="absmiddle" style="padding-left:2px"></a></li><li class="list-group-item">
                    Kopp, J., Schwede, T. The SWISS-MODEL Repository of
                    annotated three-dimensional protein structure homology
                    models. Nucleic Acids Res. 32, D230–D234 (2004).
                    <a href="https://pubmed.ncbi.nlm.nih.gov/14681401" target="_blank"><img src="../static/img/pubmed.png" width="17" height="16" align="absmiddle"></a><a href="https://dx.doi.org/10.1093/nar/gkh008" target="_blank"><img src="../static/css/doi.jpg" width="30" height="16" align="absmiddle" style="padding-left:2px"></a></li><li class="list-group-item">
                    Schwede, T., Kopp, J., Guex, N., Peitsch, M.C. SWISS-MODEL:
                    an automated protein homology-modeling server. Nucleic
                    Acids Res. 31, 3381-3385 (2003).
                    <a href="https://pubmed.ncbi.nlm.nih.gov/12824332" target="_blank"><img src="../static/img/pubmed.png" width="17" height="16" align="absmiddle"></a><a href="https://dx.doi.org/10.1093/nar/gkg520" target="_blank"><img src="../static/css/doi.jpg" width="30" height="16" align="absmiddle" style="padding-left:2px"></a></li><li class="list-group-item">
                    Guex, N., Diemand, A., Peitsch, M.C. Protein modelling for
                    all. Trends Biochem. Sci. 24, 364 - 367 (1999).
                    <a href="https://pubmed.ncbi.nlm.nih.gov/10470037" target="_blank"><img src="../static/img/pubmed.png" width="17" height="16" align="absmiddle"></a><a href="https://dx.doi.org/10.1016/S0968-0004(99)01427-9" target="_blank"><img src="../static/css/doi.jpg" width="30" height="16" align="absmiddle" style="padding-left:2px"></a></li><li class="list-group-item"><a id="promodII" name="promodII"></a>
                    Guex, N., Peitsch, M.C. SWISS-MODEL and the Swiss-PdbViewer:
                    an environment for comparative protein modeling.
                    Electrophoresis 18, 2714-2723 (1997).
                    <a href="https://pubmed.ncbi.nlm.nih.gov/9504803" target="_blank"><img src="../static/img/pubmed.png" width="17" height="16" align="absmiddle"></a><a href="https://dx.doi.org/10.1002/elps.1150181505" target="_blank"><img src="../static/css/doi.jpg" width="30" height="16" align="absmiddle" style="padding-left:2px"></a></li><li class="list-group-item">
                    Peitsch, M.C. ProMod and Swiss-Model: Internet-based tools
                    for automated comparative protein modelling. Biochem. Soc.
                    Trans. 24, 274-279 (1996)
                    <a href="https://pubmed.ncbi.nlm.nih.gov/8674685" target="_blank"><img src="../static/img/pubmed.png" width="17" height="16" align="absmiddle"></a><a href="https://dx.doi.org/10.1042/bst0240274" target="_blank"><img src="../static/css/doi.jpg" width="30" height="16" align="absmiddle" style="padding-left:2px"></a></li><li class="list-group-item">
                    Peitsch, M.C., Herzyk, P., Wells, T.N., Hubbard, R.E.
                    Automated modelling of the transmembrane region of
                    G-protein coupled receptor by Swiss-model. Recept. Channels
                    4, 161-4 (1996).
                    <a href="https://pubmed.ncbi.nlm.nih.gov/9014239" target="_blank"><img src="../static/img/pubmed.png" width="17" height="16" align="absmiddle"></a></li><li class="list-group-item">
                    Peitsch, M.C., Tschopp, J. Comparative molecular modelling
                    of the Fas-ligand and other members of the TNF family. Mol.
                    Immunol. 32, 761-772 (1995).
                    <a href="https://pubmed.ncbi.nlm.nih.gov/7544870" target="_blank"><img src="../static/img/pubmed.png" width="17" height="16" align="absmiddle"></a><a href="https://dx.doi.org/10.1016/0161-5890(95)00016-8" target="_blank"><img src="../static/css/doi.jpg" width="30" height="16" align="absmiddle" style="padding-left:2px"></a></li><li class="list-group-item">
                    Peitsch, M.C. Protein Modeling by E-mail. Biotechnology
                    (N.Y.) 13, 658–660 (1995).
                    <a href="https://dx.doi.org/10.1038/nbt0795-658" target="_blank"><img src="../static/css/doi.jpg" width="30" height="16" align="absmiddle" style="padding-left:2px"></a></li><li class="list-group-item">
                    Peitsch, M.C., Jongeneel, C.V. A 3-D model for the CD40
                    ligand predicts that it is a compact trimer similar to the
                    tumor necrosis factors. Int. Immunol. 5, 233-238 (1993).
                    <a href="https://pubmed.ncbi.nlm.nih.gov/8095800" target="_blank"><img src="../static/img/pubmed.png" width="17" height="16" align="absmiddle"></a></li></ul></div></div></div></li></ul><hr style="margin:32px"><h3>SWISS-MODEL would not be possible without the following resources and tools</h3><ul><li><a name="blastp"></a><b>BLAST</b><br>
  Camacho, C., Coulouris, G., Avagyan, V., Ma, N., Papadopoulos, J., Bealer, K.
  and Madden, T.L. (2009) BLAST+: architecture and applications. BMC
  Bioinformatics, 10, 421-430. <a target="_blank" href="https://pubmed.ncbi.nlm.nih.gov/20003500"><img width="17" align="absmiddle" height="16" src="../static/img/pubmed.png"></a></li><li><a name="blosum62"></a><b>BLOSUM62</b></li>
  Henikoff S. and Henikoff J. G. (1992) Amino acid substitution matrices from protein blocks. Proc Natl Acad Sci USA, 89(22): 10915-10919.
  <a target="_blank" href="https://pubmed.ncbi.nlm.nih.gov/1438297"><img width="17" align="absmiddle" height="16" src="../static/img/pubmed.png"></a><li><a name="cameo"></a><b>CAMEO - Continuous Automated Model EvaluatiOn</b><br>
            Haas, J., Barbato, A., Behringer, D., Studer, G., Roth, S.,
            Bertoni, M., Mostaguir, K., Gumienny, R., Schwede, T. (2017)
            Continuous Automated Model EvaluatiOn (CAMEO) complementing the
            critical assessment of structure prediction in CASP12. Proteins.
            <a target="_blank" href="https://pubmed.ncbi.nlm.nih.gov/29178137"><img width="17" align="absmiddle" height="16" src="../static/img/pubmed.png"></a></li><li><a name="cdhit"></a><b>CD-HIT</b><br>Li, W., L., Jaroszewski, Godzik, A. (2002). "Sequence
          clustering strategies improve remote homology recognitions while reducing
          search times." Protein Eng 15(8): 643-649.
          <a target="_blank" href="https://pubmed.ncbi.nlm.nih.gov/12364578"><img width="17" align="absmiddle" height="16" src="../static/img/pubmed.png"></a></li><li><a name="clustalw"></a><b>ClustalW2</b><br> Larkin MA, Blackshields G, Brown NP, Chenna R, McGettigan PA, McWilliam H, Valentin F, Wallace IM, Wilm A, Lopez R, Thompson JD, Gibson TJ, Higgins DG.
          (2007). Clustal W and Clustal X version 2.0. Bioinformatics, 23, 2947-2948.
          <a target="_blank" href="https://pubmed.ncbi.nlm.nih.gov/17846036"><img width="17" align="absmiddle" height="16" src="../static/img/pubmed.png"></a></li><li><a name="charmm22cmap_ref"></a><b>CHARMM22/CMAP</b><br> Mackerell AD Jr, Feig M., Brooks CL 3rd. (2004). "Extending the treatment of backbone energetics in protein force fields: limitations of gas-phase quantum mechanics in reproducing protein conformational distributions in molecular dynamics simulations." J Comput Chem 25(11):1400-15.
          <a target="_blank" href="https://pubmed.ncbi.nlm.nih.gov/15185334"><img width="17" align="absmiddle" height="16" src="../static/img/pubmed.png"></a></li><li><a name="dssp"></a><b>DSSP</b><br>Kabsch, W. and C. Sander (1983). "Dictionary of
          protein secondary structure: pattern recognition of hydrogen-bonded and
          geometrical features." Biopolymers 22: 2577-2637.
          <a target="_blank" href="https://pubmed.ncbi.nlm.nih.gov/6667333"><img width="17" align="absmiddle" height="16" src="../static/img/pubmed.png"></a></li><li><b>Graphviz</b><br> Ellson J., Gansner E., Koutsofios L., North S.C., Woodhull G.
  (2002) Graphviz— Open Source Graph Drawing Tools. In: Mutzel P., Jünger M.,
  Leipert S. (eds) Graph Drawing. GD 2001. Lecture Notes in Computer Science,
  vol 2265. Springer, Berlin, Heidelberg
</li><a href="https://dx.doi.org/10.1007/3-540-45848-4_57" target="_blank"><img src="../static/css/doi.jpg" width="30" height="16" align="absmiddle" style="padding-left:2px"></a><li><b>GISAID</b><br> Elbe S., and Buckland-Merrett G. (2017) "Data, disease and diplomacy:
  GISAID’s innovative contribution to global health." Global Challenges 1: 33-46
  <a href="https://pubmed.ncbi.nlm.nih.gov/31565258" target="_blank"><img src="../static/img/pubmed.png" width="17" height="16" align="absmiddle"></a><a href="https://dx.doi.org/10.1002/gch2.1018" target="_blank"><img src="../static/css/doi.jpg" width="30" height="16" align="absmiddle" style="padding-left:2px"></a></li><li><a name="gromos"></a><b>Gromos</b><br>van Gunsteren, W. F. (1996).
          Biomolecular Simulations: The GROMOS96 Manual and User Guide. Zürich,
          VdF Hochschulverlag ETHZ. <br></li><li><a name="hhblits"></a><b>HHblits</b><br>Steinegger, M., Meier, M., Mirdita, M., Vöhringer, H., Haunsberger, S. J., Söding, J. (2019). "HH-suite3 for fast remote homology detection and deep protein annotation.", BMC Bioinformatics 20, 473.
        <a target="_blank" href="https://pubmed.ncbi.nlm.nih.gov/31521110"><img width="17" align="absmiddle" height="16" src="../static/img/pubmed.png"></a></li><li><b>HighCharts</b><br>JavaScript library http://www.highcharts.com</li><li><a name="iprscan"></a><b>InterPro</b><br>
  Jones P., Binns D., Chang H., Fraser M, Li W, McAnulla C, McWilliam H, Maslen J,
  Mitchell A., Nuka G., Pesseat S., Quinn A.F., Sangrador-Vegas A., Scheremetjew M.,
  Yong S., Lopez R., and Hunter S. (2014). InterProScan 5: genome-scale protein
  function classification. Bioinformatics, Jan 2014.
  <a href="https://dx.doi.org/10.1093/bioinformatics/btu031" target="_blank"><img src="../static/css/doi.jpg" width="30" height="16" align="absmiddle" style="padding-left:2px"></a></li><li><b>jQuery</b><br>JavaScript Library https://jquery.com/</li><li><b>Matplotlib</b><br>
        Hunter, J. D. (2007). "Matplotlib: A 2D graphics environment", Comput Sci Eng 9, 90-95.
        <a href="https://dx.doi.org/10.1109/MCSE.2007.55" target="_blank"><img src="../static/css/doi.jpg" width="30" height="16" align="absmiddle" style="padding-left:2px"></a></li><li><a name="proteinsInMembranes"></a><b>Membrane Prediction</b><br>
Lomize, A.L., Pogozheva I.D., Lomize M.A., Mosberg H.I. (2006). Positioning of proteins in membranes: A computational approach. Protein Sci. 
  <a href="https://pubmed.ncbi.nlm.nih.gov/16731967" target="_blank"><img src="../static/img/pubmed.png" width="17" height="16" align="absmiddle"></a><a href="https://doi.org/10.1110/ps.062126106" target="_blank"><img src="../static/css/doi.jpg" width="30" height="16" align="absmiddle" style="padding-left:2px"></a></li><li><a name="molprobity"></a><b>MolProbity:</b><br>
Chenn V.B., Arendall W.B., Headd J.J., Keedy D.A., Immormino R.M., Kapral G.J., Murray L.W., Richardson J.S., and Richardson D.C. Acta Cryst. D66, 16-21 (2010). All-atom structure validation for macromolecular crystallography. 
<a target="_blank" href="https://dx.doi.org/10.1107/S0907444909042073"><img width="30" align="absmiddle" height="16" src="../static/css/doi.jpg"></a><a target="_blank" href="https://pubmed.ncbi.nlm.nih.gov/20057044"><img width="17" align="absmiddle" height="16" src="../static/img/pubmed.png" style="padding-left:2px"></a></li><li><b>Molscript:</b><br>
          Per J. Kraulis (1991) MOLSCRIPT: A Program to Produce Both Detailed
          and Schematic Plots of Protein Structures, Journal of Applied Crystallography<b>24</b>, 946-950.
        <a href="https://dx.doi.org/10.1107/S0021889891004399" target="_blank"><img src="../static/css/doi.jpg" width="30" height="16" align="absmiddle"></a></li><li><b>NGL</b><br><a name="ngl"></a>
    AS Rose, AR Bradley, Y Valasatava, JM Duarte, A Prlić and PW Rose. Web-based molecular graphics for large complexes. ACM Proceedings of the 21st International Conference on Web3D Technology (Web3D '16): 185-186, 2016.
    <a href="https://dx.doi.org/10.1145/2945292.2945324" target="_blank"><img src="../static/css/doi.jpg" width="30" height="16" align="absmiddle" style="padding-left:2px"></a><br>
    AS Rose and PW Hildebrand. NGL Viewer: a web application for molecular visualization. Nucl Acids Res (1 July 2015) 43 (W1): W576-W579 first published online April 29, 2015.
    <a href="https://dx.doi.org/10.1093/nar/gkv402" target="_blank"><img src="../static/css/doi.jpg" width="30" height="16" align="absmiddle" style="padding-left:2px"></a></li><li><a name="openmm"></a><b>OpenMM</b><br>Eastman P., Swails J., Chodera J. D., McGibbon R. T., Zhao Y., Beauchamp K. A.,
Wang L. P., Simmonett A. C., Harrigan M. P., Stern C. D., Wiewiora R. P., Brooks B. R.,
Pande V. S. (2017). "OpenMM 7: Rapid Development of High Performance Algorithms
for Molecular Dynamics." PLoS Comput. Biol. 2017.
<a target="_blank" href="https://dx.doi.org/10.1371/journal.pcbi.1005659"><img width="30" align="absmiddle" height="16" style="padding-left:2px" src="../static/css/doi.jpg"></a><a target="_blank" href="https://pubmed.ncbi.nlm.nih.gov/28746339"><img width="17" align="absmiddle" height="16" src="../static/img/pubmed.png" style="padding-left:2px"></a></li><li><a name="ost"></a><b>OpenStructure (OST)</b><br>Biasini M., Schmidt T., Bienert S., Mariani V., Studer G., Haas J., Johner N., Schenk A.D., Philippsen A. and Schwede T. (2013) "OpenStructure: an integrated software framework for computational structural biology."           Acta Cryst 2013.           <a target="_blank" href="https://dx.doi.org/10.1107/S0907444913007051"><img width="30" align="absmiddle" height="16" style="padding-left:2px" src="../static/css/doi.jpg"></a><a target="_blank" href="https://pubmed.ncbi.nlm.nih.gov/23633579"><img width="17" align="absmiddle" height="16" src="../static/img/pubmed.png" style="padding-left:2px"></a></li><li><b>PDB</b><br><a name="pdb"></a>Berman, H., Westbrook J., Feng Z., Gilliland G., Bhat T. N., Weissig I., Shindyalov I., Bourne P. E.  (2000) "The Proten Data Bank". Nucleic Acids Res, 28, 235-242.           <a target="_blank" href="https://pubmed.ncbi.nlm.nih.gov/12037327"><img width="17" align="absmiddle" height="16" src="../static/img/pubmed.png"></a><br>
Berman, H., Henrick, K., Nakamura, H. and Markley, J.L. (2007) The worldwide Protein Data Bank (wwPDB): ensuring a single, uniform archive of PDB data. Nucleic Acids Res, 35, D301-303.
          <a target="_blank" href="https://pubmed.ncbi.nlm.nih.gov/17142228"><img width="17" align="absmiddle" height="16" src="../static/img/pubmed.png"></a></li><li><a id="pisces"></a><b>PISCES</b><br>
  Wang G, Dunbrack RL Jr.
  (2003) PISCES: A protein sequence culling server. Bioinformatics 19:1589–1591.(2016) 
  <a target="_blank" href="https://pubmed.ncbi.nlm.nih.gov/12912846"><img width="17" align="absmiddle" height="16" src="../static/img/pubmed.png"></a></li><li><a id="pfam"></a><b>PFAM</b><br>
  R.D. Finn, P. Coggill, R.Y. Eberhardt, S.R. Eddy, J. Mistry, A.L. Mitchell,
  S.C. Potter, M. Punta, M. Qureshi, A. Sangrador-Vegas, G.A. Salazar,
  J. Tate, A. Bateman (2016) The Pfam protein families database: towards a
  more sustainable future. Nucleic Acids Research (2016)  Database Issue 44:D279-D285
  <a href="https://dx.doi.org/10.1093/nar/gkv1344" target="_blank"><img src="../static/css/doi.jpg" width="30" height="16" align="absmiddle" style="padding-left:2px"></a></li><li><a id="plip_refs"></a><b>PLIP</b><br>
  S. Salentin, S. Schreiber, V.J. Haupt, M.F. Adasme, M. Schroeder (2015)
  PLIP: fully automated protein–ligand interaction profiler. Nucleic Acids Research 
  43 (Web Server issue):W443-W447. doi:10.1093/nar/gkv315.
  <a href="https://dx.doi.org/10.1093/nar/gkv315" target="_blank"><img src="../static/css/doi.jpg" width="30" height="16" align="absmiddle" style="padding-left:2px"></a></li><li><a name="psipred"></a><b>PsiPred</b><br>Jones, D. T. (1999). "Protein secondary
          structure prediction based on position-specific scoring matrices."
          J Mol Biol 292(2): 195-202.
          <a target="_blank" href="https://pubmed.ncbi.nlm.nih.gov/10493868"><img width="17" align="absmiddle" height="16" src="../static/img/pubmed.png"></a></li><li><a name="pv"></a><b>PV</b><br>Biasini, M. (2014). "PV - WebGL-based Protein Viewer."
          <a href="https://dx.doi.org/10.5281/zenodo.12620" target="_blank"><img src="../static/css/doi.jpg" width="30" height="16" align="absmiddle" style="padding-left:2px"></a></li><li><a id="interpro"></a><b>InterPro</b><br><a name="interpro"></a>Mulder,  N. J., Apweiler, R., Attwood, T. K., Bairoch, A., Bateman, A., Binns, D., Bradley, P., Bork, P., Bucher, P., Cerutti, L., Copley, R., Courcelle, E., Das, U., Durbin, R., Fleischmann, W., Gough, J., Haft, D., Harte, N., Hulo, N., Kahn, D., Kanapin, A., Krestyaninova, M., Lonsdale, D., Lopez, R., Letunic, I., Madera, M., Maslen, J., McDowall, J., Mitchell, A., Nikolskaya, A. N., Orchard, S., Pagni, M., Ponting, C. P., Quevillon, E., Selengut, J., Sigrist, C. J., Silventoinen, V., Studholme, D. J., Vaughan, R., Wu, C. H. (2005). "InterPro,
          progress and status in 2005." Nucleic Acids Res 33 Database Issue:
          D201-205.
          <a target="_blank" href="https://pubmed.ncbi.nlm.nih.gov/15608177"><img width="17" align="absmiddle" height="16" src="../static/img/pubmed.png"></a></li><li><b>Raphaël JS</b><br>JavaScript Library http://dmitrybaranovskiy.github.io/raphael/</li><li><b>Raster3D</b><br><a name="raster3D"></a>Merritt E. A., Bacon D. J. (1997). "Raster3D Photorealistic Molecular Graphics." Methods in Enzymology 1997, 277 505-524      <a target="_blank" href="https://pubmed.ncbi.nlm.nih.gov/18488322"><img width="17" align="absmiddle" height="16" src="../static/img/pubmed.png" style="padding-left:2px"></a></li><li><a name="scrwl4"></a><b>SCRWL4</b><br>Krivov G. G., Shapovalov M. V., Dunbrack R. L. Jr. (2009). "Improved prediction of protein side-chain conformations with SCWRL4." Proteins 2009.           <a target="_blank" href="https://dx.doi.org/10.1002/prot.22488"><img width="30" align="absmiddle" height="16" style="padding-left:2px" src="../static/css/doi.jpg"></a><a target="_blank" href="https://pubmed.ncbi.nlm.nih.gov/19603484"><img width="17" align="absmiddle" height="16" src="../static/img/pubmed.png" style="padding-left:2px"></a></li><li><a name="sifts"></a><b>SIFTS</b><br>
Velankar, S., Dana, J.M., Jacobsen, J., van Ginkel, G., Gane, P.J., Luo, J., Oldfield, T.J., O'Donovan, C., Martin, M.J. and Kleywegt, G.J. (2013) "SIFTS: Structure Integration with Function, Taxonomy and
Sequences resource." Nucleic acids research, 41, D483-489. <a target="_blank" href="https://pubmed.ncbi.nlm.nih.gov/23203869"><img width="17" align="absmiddle" height="16" src="../static/img/pubmed.png"></a></li><li><b>SSpro / ACCpro</b><br>
          Cheng J, Randall AZ, Sweredoski MJ, Baldi P. (2005) SCRATCH: a Protein Structure and Structural Feature Prediction Server, Nucleic Acids Research, vol. 33 (web server issue), w72-76, 2005.
          <a target="_blank" href="https://pubmed.ncbi.nlm.nih.gov/15980571"><img width="17" align="absmiddle" height="16" src="../static/img/pubmed.png"></a></li><li><b>STRING</b><br>
Szklarczyk D, Franceschini A, Wyder S, Forslund K, Heller D, Huerta-Cepas J, Simonovic M, Roth A, Santos A, Tsafou KP, Kuhn M, Bork P, Jensen LJ, von Mering C. (2015). "STRING v10: protein-protein interaction networks, integrated over the tree of life." Nucleic Acids Res, 43, D447-52.
<a target="_blank" href="https://pubmed.ncbi.nlm.nih.gov/25352553"><img width="17" align="absmiddle" height="16" src="../static/img/pubmed.png"></a></li><li><a name="swissdock_refs"></a><b>SwissDock</b><br>
Grosdidier A, Zoete V, Michielin O. (2011). "SwissDock, a protein-small molecule docking web service based on EADock DSS." Nucleic Acids Res, 39, W270-7
<a target="_blank" href="https://pubmed.ncbi.nlm.nih.gov/21624888"><img width="17" align="absmiddle" height="16" src="../static/img/pubmed.png"></a><br>Grosdidier A, Zoete V, Michielin O. (2011). "Fast docking using the CHARMM force field with EADock DSS."  J Comput Chem. 32(10), 2149-59.
<a target="_blank" href="https://pubmed.ncbi.nlm.nih.gov/21541955"><img width="17" align="absmiddle" height="16" src="../static/img/pubmed.png"></a></li><li><a name="treepack_ref"></a><b>TreePack</b><br>
Xu J, Jiao F, Berger B. (2005). "A tree-decomposition approach to protein structure prediction." Proc IEEE Comput Syst Bioinform Conf, 247-56
<a target="_blank" href="https://pubmed.ncbi.nlm.nih.gov/16447982"><img width="17" align="absmiddle" height="16" src="../static/img/pubmed.png"></a></li><li><a name="uniclust"></a><b>Uniclust</b><br>Mirdita, M., von den Driesch, L., Galiez, C., Martin, M.J., Söding, J., Steinegger, M. (2016) Uniclust databases of clustered and deeply annotated protein sequences and alignments. Nucleic Acids Res, 45, D170–D176.
  <a target="_blank" href="https://dx.doi.org/10.1093/nar/gkw1081"><img width="30" align="absmiddle" height="16" style="padding-left:2px" src="../static/css/doi.jpg"></a><a target="_blank" href="https://pubmed.ncbi.nlm.nih.gov/27899574"><img width="17" align="absmiddle" height="16" src="../static/img/pubmed.png"></a></li><li><a name="uniprot"></a><b>UniProtKB</b><br>The UniProt Consortium. (2017) UniProt: the universal protein knowledgebase. Nucleic Acids Res, 45, D158-D169.
  <a target="_blank" href="https://pubmed.ncbi.nlm.nih.gov/27899622"><img width="17" align="absmiddle" height="16" src="../static/img/pubmed.png"></a></li></ul></div><div id="loginForm" style="width:600px;display:none;padding-top:12px; text-align:center;"></div><footer class="footer hidden-print"><div id="sib_footer"><ul class="list-inline" style="margin-bottom:0px; padding-bottom:4px;padding-top:4px;"><li><a href="https://www.sib.swiss/">Swiss Institute of Bioinformatics</a></li><li><a href="https://www.biozentrum.unibas.ch/research/researchgroups/overview/unit/schwede/">Schwede Group</a></li><li><a href="mailto:mouseover@for.address" class="swissmodelContactRef">Contact Us</a></li><li><a href="https://swissmodel.expasy.org/docs/terms_of_use">Terms of use</a></li><li class="pull-right"><a id="sib_footer_gototop" onclick="window.scrollTo(0,0);"><i class="glyphicon glyphicon-arrow-up"></i> Back to the Top</a></li></ul><div class="clearfix"></div></div></footer><script src="../static/js/jquery-3.4.1.min.js"></script><script src="../static/js/jquery-ui.min.js"></script><script src="../static/js/bootstrap-3.3.7.min.js"></script><script src="../static/js/jquery.fancybox.pack.js"></script><script>
    $('a.smng-help').append('<icon class="glyphicon glyphicon-question-sign"></i>');
    $(document).on('click','a.smng-help',function()
    { 
        window.open($(this).attr('href'),
            'Help',
            'width=920,height=600,left=20,top=100,directories=no,menubar=yes,toolbar=yes,'
                +'status=no,scrollbars=yes,resizable=no');
        return false;
    });

     $('a.swissmodelContactRef')
        .on('mouseover',function()
            {
                $(this).attr('href',["ch", ".", "nibas", "l@u",
             "mode", "-swiss",
              ":help", "to", "ail", "m"].reverse().join(''));
                $(this).off('mouseover');
            });

     function showLogin()
     {
        $('#loginForm')
            .load('/login', { 'csrfmiddlewaretoken':'VldqXO7mYKCZi4qGaLd22YLrsMNY3V5PiL3rEWFpMbouPXX82AUEoMNi4Yo6xmGF' },
                function()
                {
                    $("<a href='#loginForm'/>'").fancybox().trigger('click');  
                    $('#id_username').focus();
                });
        
     } 
     function createAccount()
     {
        $('#loginForm')
            .load('/create_account',
                function()
                {
                    $("<a href='#loginForm'/>'").fancybox().trigger('click'); 
                    $('#id_username').focus();  
                });
     }
     
</script>
</body></html>