<!DOCTYPE html>
<!-- saved from url=(0050)https://swissmodel.expasy.org/docs/repository_help -->
<html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><!-- Global site tag (gtag.js) - Google Analytics --><script type="text/javascript" async="" src="../static/js/analytics.js"></script><script async="" src="../static/js/js"></script><script>
      window.dataLayer = window.dataLayer || [];
      function gtag(){dataLayer.push(arguments);}
      gtag('js', new Date());

      gtag('config', 'UA-8667194-2');
    </script><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width, initial-scale=1"><meta name="ROBOTS" content="ALL"><title>Repository Help | SWISS-MODEL</title><meta name="description" content="SWISS-MODEL is a fully automated protein structure homology-modelling server. The purpose of this server is to make protein modelling accessible to all life science researchers worldwide."><link rel="stylesheet" type="text/css" href="../static/css/bootstrap-3.3.7.min.css" media="screen,print"><link rel="stylesheet" type="text/css" href="../static/css/bootstrap-theme-3.3.7.min.css" media="screen,print"><link rel="stylesheet" type="text/css" href="../static/css/master.css" media="all"><link rel="stylesheet" type="text/css" href="../static/css/jquery.fancybox.css"><link rel="stylesheet" type="text/css" href="../static/css/jquery-ui.min.css"><link rel="shortcut icon" href="https://swissmodel.expasy.org/static/images/favicon.png" type="image/png"><link rel="apple-touch-icon" href="https://swissmodel.expasy.org/static/images/apple-touch-57.png"><link rel="apple-touch-icon" sizes="72x72" href="https://swissmodel.expasy.org/static/images/apple-touch-72.png"><link rel="apple-touch-icon" sizes="114x114" href="https://swissmodel.expasy.org/static/images/apple-touch-114.png"><style> h4{ margin-top:32px;} </style></head><body><nav class="navbar navbar-default"><div class="container-fluid"><div class="navbar-header"><button aria-controls="navbar" aria-expanded="false" data-target=".collapseThis" data-toggle="collapse" class="navbar-toggle collapsed" type="button"><span class="sr-only">Toggle navigation</span><span class="icon-bar"></span><span class="icon-bar"></span><span class="icon-bar"></span></button><a class="hidden-xs hidden-sm navbar-brand" style="margin-left:0px;padding:8px 4px 0px 0px;" title="SIB Swiss Institute of Bioinformatics" href="https://www.sib.swiss/" target="_blank"><img src="../static/img/sib_logo_small.png" alt="SIB logo" style="max-height:50px"></a><a class="hidden-xs hidden-sm navbar-brand" style="margin-left:0px; padding:8px 0px 0px 4px;" title="Biozentrum | University of Basel | The Center for Molecular Life Sciences" href="http://www.biozentrum.unibas.ch/" target="_blank"><img src="../static/img/biozentrum_en.png" alt="Biozentrum logo" style="max-height:36px"></a><a class="hidden-xs hidden-sm navbar-brand" href="https://swissmodel.expasy.org/" style="margin-left:0px;padding-left:0px;padding-right:0px;font-size:28px;">SWISS-MODEL</a><a id="navbar-brand" class="hidden-md hidden-lg navbar-brand" href="https://swissmodel.expasy.org/" style="margin-left:0px;font-size:20px;">SWISS-MODEL</a></div><div id="navbar" class="collapseThis navbar-collapse collapse"><!-- Collect the nav links, forms, and other content for toggling --><ul class="nav navbar-nav navbar-right"><li class="dropdown "><a href="https://swissmodel.expasy.org/interactive" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Modelling</a><ul class="dropdown-menu" role="menu"><li><a class="text-left" href="https://swissmodel.expasy.org/interactive">myWorkspace</a></li><li><a class="text-left" href="https://swissmodel.expasy.org/interactive#alignment">Alignment Mode</a></li><li><a class="text-left" href="https://swissmodel.expasy.org/interactive#structure">User Template Mode</a></li><li><a class="text-left" href="https://swissmodel.expasy.org/interactive#deepview">DeepView Project Mode</a></li><li class="divider"></li><li><a class="text-left" href="https://swissmodel.expasy.org/templates/">Template Library</a></li></ul></li><li id="smrLink" class="dropdown "><a href="https://swissmodel.expasy.org/repository" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Repository</a><ul class="dropdown-menu" role="menu"><li><a href="https://swissmodel.expasy.org/repository">SWISS-MODEL Repository</a></li><li><a href="https://swissmodel.expasy.org/repository/user_annotation_upload">Annotation Projects</a></li></ul></li><li class="dropdown "><a class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Tools</a><ul class="dropdown-menu" role="menu"><li><a href="https://swissmodel.expasy.org/qmean/" tabindex="-1">QMEAN</a></li><li><a href="https://swissmodel.expasy.org/lddt" tabindex="-1">lDDT</a></li><li><a href="https://swissmodel.expasy.org/assess">Structure Assessment</a></li><li><a href="https://swissmodel.expasy.org/comparison/">Structure Comparison</a></li></ul></li><li class="dropdown"><a href="https://swissmodel.expasy.org/docs/repository_help#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Documentation</a><ul class="dropdown-menu" role="menu"><li><a href="https://swissmodel.expasy.org/docs/help" tabindex="-1">SWISS-MODEL</a></li><li><a href="https://swissmodel.expasy.org/docs/examples" tabindex="-1">Examples</a></li><li><a href="https://swissmodel.expasy.org/docs/tutorial" tabindex="-1">Video Tutorial</a></li><li><a href="https://swissmodel.expasy.org/docs/blog" tabindex="-1">New Features</a></li><li role="separator" class="divider"></li><li><a href="https://swissmodel.expasy.org/docs/repository_help" tabindex="-1">SWISS-MODEL Repository</a></li><li role="separator" class="divider"></li><li><a href="https://swissmodel.expasy.org/qmean/help" tabindex="-1">QMEAN</a></li><li><a href="https://swissmodel.expasy.org/lddt/help/" tabindex="-1">lDDT</a></li><li><a href="https://swissmodel.expasy.org/assess/help">Structure Assessment</a></li><li><a href="https://swissmodel.expasy.org/comparison/help">Structure Comparison</a></li><li role="separator" class="divider"></li><li><a href="https://swissmodel.expasy.org/course" tabindex="-1">Protein Structure Course</a></li><li role="separator" class="divider"></li><li><a href="https://swissmodel.expasy.org/docs/references" tabindex="-1">References</a></li></ul></li><li><a onclick="showLogin();">Log in</a></li><li><a onclick="createAccount();">Create Account</a></li></ul></div></div><!-- /container-fluid --></nav><div id="tools"></div><div id="main-content" style="padding-left:32px; margin-right:32px"><div class="row docu"><div class="col-md-3 col-md-push-9 docu-md-fix"><nav class="docu_nav"><ul class="nav"><li class=""><a href="https://swissmodel.expasy.org/docs/repository_help#intro_smr" class="docu_link active">Introduction to SWISS-MODEL Repository </a></li><li><a href="https://swissmodel.expasy.org/docs/repository_help#smr_cont" class="docu_link">Data content and update</a></li><li><a href="https://swissmodel.expasy.org/docs/repository_help#smr_data_acces" class="docu_link">Accessing data</a><ul class="nav"><li><a href="https://swissmodel.expasy.org/docs/repository_help#smr_species_page" class="docu_link">Model species page</a></li><li><a href="https://swissmodel.expasy.org/docs/repository_help#smr_entry" class="docu_link">SWISS-MODEL Repository entry page</a></li><ul class="nav"><li><a href="https://swissmodel.expasy.org/docs/repository_help#smr_coverage" class="docu_link">Structural coverage</a></li><li><a href="https://swissmodel.expasy.org/docs/repository_help#smr_summary" class="docu_link">Entry summary</a></li><li><a href="https://swissmodel.expasy.org/docs/repository_help#smr_visualization" class="docu_link">3D structure visualisation</a></li><li><a href="https://swissmodel.expasy.org/docs/repository_help#smr_alignment" class="docu_link">Targetâ€“template sequence alignment</a></li><li><a href="https://swissmodel.expasy.org/docs/repository_help#smr_parameters" class="docu_link">Custom parameters</a></li></ul></ul></li><li><a href="https://swissmodel.expasy.org/docs/repository_help#smr_quality" class="docu_link">Model quality</a></li><li><a href="https://swissmodel.expasy.org/docs/repository_help#smr_update" class="docu_link">On-demand update of individual entries</a></li><li><a href="https://swissmodel.expasy.org/docs/repository_help#smr_api" class="docu_link">API</a></li></ul></nav></div><div class="col-md-9 col-md-pull-3"><h1>SWISS-MODEL Repository Help</h1><div id="intro_smr"><h2>Introduction to SWISS-MODEL Repository</h2><p>
         The <a href="https://swissmodel.expasy.org/repository/">SWISS-MODEL Repository (SMR)</a>
         is a database of 3D protein structure models generated by the SWISS-MODEL
         homology modelling pipeline. It provides access to an up-to-date collection
         of annotated 3D protein models generated by automated homology modelling for
         all sequences in Swiss-Prot and for relevant model organisms. Currently,
         models are provided for the reference proteomes of 12 model organisms.
         Regular updates ensure that target coverage is
         complete, that models are built using the most recent sequence and template
         structure databases, and that improvements in the underlying modelling pipeline
         is fully utilised.
      </p><p>
        Users can assess the quality of the models in the database, search for alternative
        template structures, and build models interactively via <a href="https://swissmodel.expasy.org/interactive">
        SWISS-MODEL Workspace</a>. Annotation of models with functional information
        and cross-linking with other databases, such as <a href="https://www.uniprot.org/" target="_blank">UniProtKB</a>, facilitates the navigation
        between protein sequence and structure resources.
      </p><p><b>If you are using models from the <i>SWISS-MODEL Repository</i>, please cite the following articles:</b></p><p>Bienert, S., Waterhouse, A., de Beer, T.A.P., Tauriello, G., Studer, G.,
        Bordoli, L., Schwede, T. The SWISS-MODEL Repository - new features and
        functionality. Nucleic Acids Res. 45, D313-D319 (2017).
              <a href="https://pubmed.ncbi.nlm.nih.gov/27899672" target="_blank"><img src="../static/img/pubmed.png" width="17" height="16" align="absmiddle"></a><a href="https://dx.doi.org/10.1093/nar/gkw1132" target="_blank"><img src="../static/img/doi.jpg" width="30" height="16" align="absmiddle" style="padding-left:2px"></a></p><p>Waterhouse, A., Bertoni, M., Bienert, S., Studer, G., Tauriello, G., Gumienny, R.,
          Heer, F.T., de Beer, T.A.P., Rempfer, C., Bordoli, L., Lepore, R., Schwede, T.   
          SWISS-MODEL: homology modelling of protein structures and complexes.
          Nucleic Acids Res. 46(W1), W296-W303 (2018).
          <a href="https://pubmed.ncbi.nlm.nih.gov/29788355" target="_blank"><img src="../static/img/pubmed.png" width="17" height="16" align="absmiddle"></a><a href="https://doi.org/10.1093/nar/gky427" target="_blank"><img src="../static/img/doi.jpg" width="30" height="16" align="absmiddle" style="padding-left:2px"></a></p><br></div><div id="smr_cont"><h2>Data content and update</h2><p>SMR currently provide models for the Swiss-Prot section of UniProtKB and for
        the reference proteomes of the following model organisms: <i>H. sapiens, M. musculus,
        C. elegans, E. coli K12, A. thaliana, D. melanogaster, S. cerevisiae,
        Caulobacter crescentus, M. tuberculosis, P. aeruginosa, Staphylococcus aureus</i>,
       and <i>P. falciparum</i>. </p><p>For each core species proteome, SMR provides a dedicated summary page with
        information on the target proteome, statistics on model coverage as well as
        the analysis of structural coverage of the proteome over time.</p><p>The full set of model data for each core species is downloadable from the main
        entry page as well as the summary page. Models for non-core sequences are added
        on demand by interactive user request.</p><p>A continuous release mechanism ensures that the models are regularly updated
        based on the latest template information and the latest version of the
        SWISS-MODEL modelling pipeline. The current version of the SMR features a
        three-tiered policy for updating: monthly, weekly, and on-demand. </p><p>Monthly updates are initiated when a new version of UniProtKB is released,
        i.e. the amino acid sequences of UniProtKB entries may have changed or the
        composition of the core species proteomes may have been updated.</p><p>Weekly updates ensure that the latest PDB release is readily integrated in the
        SWISS MODEL Template Library (SMTL), and models are built using the latest
        template information.</p><p>On-demand updates are meant for sequences that are not part of the core species
        data set. When accessing an SMR entry for which no models have been built,
        or the existing models are not based on current template information, the user
        is presented with two options: (i) requesting an automated update of the SMR
        entry (ii) starting an interactive modelling job on the SWISS-MODEL Workspace.</p><p id="smrQualityThreshold">Note: <b>Only models of high quality are imported into SMR!</b> For models with less
      than 100 residues, the sequence identity must be over 30%. For models with greater than
      100 residues the QMEAN4 score must be greater than -5. 

      </p><br></div><div id="smr_data_acces"><h2>Accessing data</h2><p>The SMR web interface provides several entry points for accessing the data.
        The simplest way to directly access an entry is via its UniProtKB accession code.
        Alternatively, free text search on protein names, functional description, and
        organisms allows selecting entries from a list of results matching the search keywords. </p><p>The main page offers a general summary about the repository content and,
        for each of the core species, the following information: total number of
        canonical sequences in the reference proteome (according to UniProtKB),
        number of sequences for which at least one model is present, total number of
        models for that species, and a sequence coverage plot showing the number of
        sequences that have models or structures in a certain sequence coverage
        range (&gt;80%, between 60% and 80%, etc.). A colour gradient, from dark-green
        to red, specifies  different coverage values, from high to low respectively.
        The size of each box is proportional to the number of target sequences within
        a given range of coverage (i.e. for <i>E. Coli</i>, more than 60% of the proteins are
        covered by more than 80% over their sequence length by structures and models).</p><p>A download button allows to download all the metadata about models and
        structures of a given core species (further details are provided in the help
        file in the downloaded folder as well as in the "SMR API" section below).</p><br><div id="smr_species_page"><h3>Model species page</h3><p>By clicking on the core species names the user can access a dedicated summary
          page, with information on the target proteome, statistics on model coverage
          as well as the evolution of structural coverage of the proteome over time.
          The full set of model data for each core species is downloadable from the main
          entry page as well as the summary page.</p><br></div><div id="smr_entry"><h3>SWISS-MODEL Repository entry page</h3><p>Individual entries can be searched via UniProtKB accession codes or using a
          free text search on protein names, functional description and/or organisms.
          In the latter case the user is provided with a list of entries matching the
          search keywords and access the information of each entry by selecting from the list.</p><p>The entry view page for a specific entry provides four general sections:
          (i) a graphical representation of the target coverage by models and experimental
          structures, (ii) a SMR entry summary, (iii) a 3D structure visualisation of the
          entry, and (iv) the targetâ€“template sequence alignment. Each of these sections
          is described below.</p><br><div id="smr_coverage"><h4>Structural coverage</h4><p>The amino acid sequence of the target is indicated as a gray arc. The information
            of the structural coverage is indicated on the inner side of the arc, where
            solid segments indicate available experimentally determined structures and
            dashed outlines indicate homology models. </p><p>Sequence annotation features are displayed on mouse-over to the outer side of
            the arc and include <a href="https://swissmodel.expasy.org/docs/references#interpro">InterPro</a> domains, <a href="https://swissmodel.expasy.org/docs/references#pfam">PFAM</a> domains, variants, transmembrane
            regions, disulphide bonds, nucleotide binding regions, signal peptides, and
            active site residues. The same information is also provided by the Sequence
            Features pull-down menu.</p><p>In this view, structures and models are grouped based on coverage and oligomeric
            state. The available models for the entry are listed at the bottom of the page. </p><br></div><div id="smr_summary"><h4>Entry summary</h4><p>The entry summary provides general information about the target protein, including
            the name of the protein and links to related information in UniProtKB,
            InterPro, and STRING databases.</p><p>For each selected model, the following information is provided: the PDB ID
            of the structure used as a template for modelling, the targetâ€“template sequence
            identity and similarity, the date of the last update of the model, the model
            quality estimate, and a link for downloading the model coordinates.</p><p>Three different model quality plots are shown, providing an estimate of
            both global and local quality of the model as assessed by the QMEAN scoring
            function, thus allowing for the identification of unreliable or poorly modelled
            regions (or even residues) of a model.</p><p>The first two plots show the global and local model quality respectively,
            while the third plot shows the QMEAN Z-score of the model in comparison
            with experimental protein structures from the PDB. The latter plot gives
            an indication on how similar a given model is in terms of mean force
            potential to experimental structures of comparable size (further details are provided in
            the <a href="https://swissmodel.expasy.org/docs/repository_help#smr_quality">Model quality</a> section below).</p><p>In case the user prefers to build a model using a different template structure
            than the one selected automatically, e.g. in a different conformation state or
            with a specific ligand, an "Interactive Modelling" button is provided with a
            link to the SWISS-MODEL workspace. Here the user can start a new interactive
            modelling session with the target sequence preloaded for initiating a new
            template selection.</p><br></div><div id="smr_visualization"><h4>3D structure visualisation</h4><p>This section provides an in-page visualisation of the currently selected
            model/structure. The graphical view has been implemented using <a href="https://biasmv.github.io/pv/">PV</a>,
            an interactive JavaScript/WebGL based 3D structure viewer, and
            <a href="https://github.com/arose/ngl">NGL</a>, a web application for molecular
            visualisation. All sections in the
            SMR entry page are linked to the 3D structure viewer. Any selected feature from
            the entry summary or the sequence alignment can be visualised on the structure
            and vice versa (i.e. a residue selected in the structure is highlight in the alignment).</p><p>Buttons are provided below the 3D viewer allowing for some extra functionalities,
            such as taking a snapshot or running PV in a separate window.</p><br></div><div id="smr_alignment"><h4>Targetâ€“template sequence alignment</h4><p>Details regarding the alignment between the target and
            the template are provided. The configuration button allows applying various colouring schemes
            to the sequence and the 3D model shown in PV. Amongst others, the user can colour
            the alignment and the structure by amino acid properties, QMEAN score, as well
            as by chain or secondary structure. An option is also provided to export the
            alignment in any of the following formats: FASTA, Clustal and PNG. </p><p>Below the alignment, a table is provided where all available experimental
            structures and models are listed, allowing an in-depth exploration of the
            available structure information for the entry.</p></div><div id="smr_parameters"><h4>Custom parameters</h4><p>The selection of the structure displayed initially, as well as several aspects
            of the initial display can be customized with URL parameters. Many of these
          can be used in combination.</p><p>An example of custom URL with combined parameters is the following: </p><p><code>https://swissmodel.expasy.org/repository/uniprot/P0DTD1?template=6m71.1.A&amp;range=4423-5311</code></p><ul class="list-group"><li class="list-group-item"><h4 class="list-group-item-heading"><small>https://swissmodel.expasy.org/repository/uniprot/P0DTD1</small>?template=6m71.1.A</h4><p class="list-group-item-text">Select the homology model based on chain A of entry
               6m71.1 of the <a href="https://swissmodel.expasy.org/templates/">SWISS MODEL Template Library (SMTL)</a>.
               If a structure from the PDB is available, the 4 alphanumeric characters of the
               PDB ID may be entered as follows:</p><h4 class="list-group-item-heading"><small>https://swissmodel.expasy.org/repository/uniprot/P0DTD1</small>?template=6w9c</h4></li><li class="list-group-item"><h4 class="list-group-item-heading"><small>https://swissmodel.expasy.org/repository/uniprot/P0DTD1</small>?range=1024-1194</h4><h4 class="list-group-item-heading"><small>https://swissmodel.expasy.org/repository/uniprot/P0DTD1</small>?from=1024&amp;to=1194</h4><p class="list-group-item-text">Select a homology model or experimental structure that covers
               residues 1024-1194 of the target sequence.</p></li><li class="list-group-item"><h4 class="list-group-item-heading"><small>https://swissmodel.expasy.org/repository/uniprot/P07900</small>?csm=969F65FCC0BC86FD</h4><p class="list-group-item-text">The CRC64 checksum of the protein sequence. This can be used to ensure the sequence for the given UniProtKB entry matches our record.</p></li><li class="list-group-item"><h4 class="list-group-item-heading"><small>https://swissmodel.expasy.org/repository/uniprot/P0DTD1</small>?id=PRO_0000449630</h4><p class="list-group-item-text">For polyproteins which cleave into smaller mature proteins, 
               the Protein ID for the given UniProtKB entry will be displayed.</p></li><li class="list-group-item"><h4 class="list-group-item-heading"><small>https://swissmodel.expasy.org/repository/uniprot/P0DTD1</small>?user-annot=3</h4><p class="list-group-item-text">If annotations have been uploaded and a project is active, display
               the annotation in row 3 (note that the numbering starts from 0). This is equivalent to clicking on
               the annotation after the page loads.</p></li><li class="list-group-item"><h4 class="list-group-item-heading"><small>https://swissmodel.expasy.org/repository/uniprot/P0DTD1</small>?annot=XXXX</h4><p class="list-group-item-text">One or more annotation in the same format as supported by
               the user <a href="https://swissmodel.expasy.org/repository/user_annotation_upload">annotation upload form</a>
               gzipped, base64-encoded and then url-encoded.
               The annotation are not stored in a project and therefore will not be available
               on other pages, or upon reload of the page without the <code>annot</code> argument.
               Some browsers may limit the total length of URL to 2000 characters.</p><p>For instance an annotation like
              <code>P0DTD1	1050	1100	#0000FF	A text reference	nsp3_232-382_covered</code> would
              be encoded as :
              </p><p style="overflow:auto;white-space: nowrap;"><code>H4sIAAAAAAAA%2FwswcAlxMeQ0NDA14DQ0NDDgVDYAAjc3TkeFktSKEoWi1LTUotS85FTOvOIC43gjYyNdYwuj%2BOT8MqBwCgC4nDDgPgAAAA%3D%3D</code></p><p></p></li></ul></div></div><br></div><div id="smr_quality"><h2>Model quality</h2><p>Each model in SMR is evaluated by QMEAN
        (<a href="https://swissmodel.expasy.org/docs/references#qmean"><i>Studer et al.</i></a>)
        to provide model quality measures on a per-residue basis as well as a global scale.
        The graphs provide plots of the estimated local quality of each part of the model
        as well as how the model compares to other structures in the PDB. In the main
        overview graph, colours are used to provide quality information at a glance
        with blue indicating good and red indicating bad quality scores for the specific
        feature. To relate the model quality to high resolution X-ray structures of
        similar size, QMEAN values are expressed as a Z-score.
        Per-residue quality values are also provided in the B-factor column of the
        PDB file of the model. More information can be found in the SWISS-MODEL
        <a target="_blank" href="https://swissmodel.expasy.org/docs/help#qmean">help</a>
        section.
        </p><br></div><div id="smr_update"><h2>On-demand update of individual entries</h2><p>If no structure or model is available for a given UniProtKB entry, the user can
        request either an automated update of the SMR entry or start an interactive
        modelling job using the SWISS-MODEL Workspace by clicking the "Build Models"
        Button.</p><p>Please notice that it may take up to 15 minutes before a model is added to the
        Repository. <br>
        Please notice that models with a low-quality estimate are not added to
        the repository.</p><br></div><div id="smr_api"><h2 id="smr_api">API <small>full details available in this <a href="https://swissmodel.expasy.org/docs/smr_openapi">OpenAPI</a> document.</small></h2><p><i>The first version of the SMR API is still currently available by adding the parameter
        ?api_version=1.0 
        <br>Please let us know if you require deprecated versions of our API to
        remain in place, otherwise we will only maintain the latest version.</i></p><p style="color:#D22">SWISS-MODEL Repository is continuously updated to provide the user with with the best available model/experimental structure for a given UniProtKB AC. Due to this continuous update, there are no fixed URLs to specific versions of models, and indeed, models may no longer exist if a better template becomes available in the SWISS-MODEL Template Library.</p><p><strong>To find out what is currently available in SMR, first get a list of
        available models and experimental structures in JSON format (here UniProtKB
        AC P07900 is used as an example).</strong></p><ul class="list-group"><li class="list-group-item"><h4 class="list-group-item-heading"><small>https://swissmodel.expasy.org/repository/uniprot/P07900</small>.json</h4><p class="list-group-item-text">Returns a JSON object, {"result":"structures":[]} is a list
              of homology models and experimental structures for UniProtKB entry P07900</p></li><li class="list-group-item"><h4 class="list-group-item-heading"><small>https://swissmodel.expasy.org/repository/uniprot/P07900</small>.json?provider=swissmodel</h4><p class="list-group-item-text">Returns a JSON object, {"result":"structures":[]} is a list of
              homology models (provider=swissmodel) for UniProtKB entry P07900</p></li><li class="list-group-item"><h4 class="list-group-item-heading"><small>https://swissmodel.expasy.org/repository/uniprot/P07900</small>.json?template=2o1u.1.A</h4><p class="list-group-item-text">Returns a JSON object, {"result":"structures":[]} is a list
              of homology models or experimental structures for the template 2o1u.1.A.
              The template ID can be indicated in one of the following formats: 2o1u, 2o1uA, 2o1u.1.A</p></li><li class="list-group-item"><h4 class="list-group-item-heading"><small>https://swissmodel.expasy.org/repository/uniprot/P07900</small>.json?range=15-250</h4><p class="list-group-item-text">Returns a JSON object, {"result":"structures":[]} is a list
              of homology models or experimental structures that have residues in range 15â€“250 of the target sequence</p></li></ul><p>From the list of available models, the user can select a specific model or
        structure based on desired parameters or a combination of them, i.e. QMEAN,
        SeqId, range. etc., and fetch the coordinates using the URL provided in the
        "coordinates" field. </p><p>As previously stated, the continuous update of SMR means a specific model may
        not exist anymore, so no specific model ID is available. However, for a given
        entry, the API will return the best model/structure available. The best model
        is the the one with the highest quality based on QMEAN. The "sort" parameter
        allows to sort the models by sequence identity or sequence similarity to
        template. Experimental structures can be sorted based on stoichiometry,
        range, experimental method, and resolution. If no provider is specified,
        experimental structures will be returned before homology models.</p><p>An example of combined URL is the following: </p><p> https://swissmodel.expasy.org/repository/uniprot/P07900.pdb?from=14&amp;to=223&amp;template=2k5b&amp;provider=pdb</p><p>If no model is found for your requested coordinates, a 404 Not Found error will be raised.</p><ul class="list-group"><li class="list-group-item"><h4 class="list-group-item-heading"><small>https://swissmodel.expasy.org/repository/uniprot/P07900</small>.pdb</h4><p class="list-group-item-text">Returns the best homology model or experimental structure in PDB format.</p></li><li class="list-group-item"><h4 class="list-group-item-heading"><small>https://swissmodel.expasy.org/repository/uniprot/P07900</small>.pdb?sort=seqsim</h4><p class="list-group-item-text">Returns the best homology model or experimental
           structure in PDB format sorted by sequence similarity.</p></li><li class="list-group-item"><h4 class="list-group-item-heading"><small>https://swissmodel.expasy.org/repository/uniprot/P07900</small>.pdb?sort=seqid</h4><p class="list-group-item-text">Returns the best homology model or experimental
           structure in PDB format sorted by sequence identity.</p></li><li class="list-group-item"><h4 class="list-group-item-heading"><small>https://swissmodel.expasy.org/repository/uniprot/P07900</small>.pdb?provider=swissmodel</h4><p class="list-group-item-text">Returns the best homology model from the given provider in PDB format.</p></li><li class="list-group-item"><h4 class="list-group-item-heading"><small>https://swissmodel.expasy.org/repository/uniprot/P07900</small>.pdb?template=2o1u.1.A</h4><p class="list-group-item-text">Returns the best homology model or experimental
           structure based on the given template in PDB format.</p></li><li class="list-group-item"><h4 class="list-group-item-heading"><small>https://swissmodel.expasy.org/repository/uniprot/P07900</small>.pdb?range=15-250</h4><p class="list-group-item-text">Returns the best homology model or
           experimental structure which overlaps the given range.</p></li></ul></div></div></div></div><div id="loginForm" style="width:600px;display:none;padding-top:12px; text-align:center;"></div><footer class="footer hidden-print"><div id="sib_footer"><ul class="list-inline" style="margin-bottom:0px; padding-bottom:4px;padding-top:4px;"><li><a href="https://www.sib.swiss/">Swiss Institute of Bioinformatics</a></li><li><a href="https://www.biozentrum.unibas.ch/research/researchgroups/overview/unit/schwede/">Schwede Group</a></li><li><a href="mailto:mouseover@for.address" class="swissmodelContactRef">Contact Us</a></li><li><a href="https://swissmodel.expasy.org/docs/terms_of_use">Terms of use</a></li><li class="pull-right"><a id="sib_footer_gototop" onclick="window.scrollTo(0,0);"><i class="glyphicon glyphicon-arrow-up"></i> Back to the Top</a></li></ul><div class="clearfix"></div></div></footer><script src="../static/js/jquery-3.4.1.min.js"></script><script src="../static/js/jquery-ui.min.js"></script><script src="../static/js/bootstrap-3.3.7.min.js"></script><script src="../static/js/jquery.fancybox.pack.js"></script><script>
function highlight(item)
{
	$('.active').removeClass('active');
	do  
	{
		item.addClass('active');
		item = item.parent()
			.parents('li:eq(0)')
			.find('a.docu_link:eq(0)');
	}
	while(item.length>0)
}

$(document).on('click','a.docu_link',function(event)
	{ 
	    $('html, body').animate(
	    	{ scrollTop: $( $.attr(this, 'href') ).offset().top }, 400,
	    	function(){ highlight( $(event.target) ); }
	    );
	});

$(document).on("scroll", function()
	{
		if($('html:animated').length>0)
			return;
		
		var scrollPos = $(document).scrollTop();
	    $('#main-content div[id]').each( function()
	    { 
	    	if($(this).offset().top>=scrollPos ) 
			{ 
				highlight( $('nav a.docu_link[href="#'+$(this).attr('id')+'"]') );
				return false 
			}
	    });
	});
</script><script>
    $('a.smng-help').append('<icon class="glyphicon glyphicon-question-sign"></i>');
    $(document).on('click','a.smng-help',function()
    { 
        window.open($(this).attr('href'),
            'Help',
            'width=920,height=600,left=20,top=100,directories=no,menubar=yes,toolbar=yes,'
                +'status=no,scrollbars=yes,resizable=no');
        return false;
    });

     $('a.swissmodelContactRef')
        .on('mouseover',function()
            {
                $(this).attr('href',["ch", ".", "nibas", "l@u",
             "mode", "-swiss",
              ":help", "to", "ail", "m"].reverse().join(''));
                $(this).off('mouseover');
            });

     function showLogin()
     {
        $('#loginForm')
            .load('/login', { 'csrfmiddlewaretoken':'Gc6DC265BDAmKfPtBfk8bmJ09d1ujTa93CWEjaE8p4mRh8mVt41KxaLRLpCCNkLZ' },
                function()
                {
                    $("<a href='#loginForm'/>'").fancybox().trigger('click');  
                    $('#id_username').focus();
                });
        
     } 
     function createAccount()
     {
        $('#loginForm')
            .load('/create_account',
                function()
                {
                    $("<a href='#loginForm'/>'").fancybox().trigger('click'); 
                    $('#id_username').focus();  
                });
     }
     
</script>
</body></html>